### DISCUSSION TITLE
Module for random integer

### DISCUSSION THREAD

**Discussion by SouljaVR** [VolcanicArts/VRCOSC#12] [Ideas]
Would be cool if there was an Integer type parameter randomizer too. Perhaps with a min/max/update rate settings. Would have lots of use cases for avatars! Thank you.

**Comment by VolcanicArts** [comment]
The random module was designed as just an example, but I'm more than happy to extend it to randomise all 3 types of VRChat parameters and have a variable rate to be of more use

**Comment by VolcanicArts** [comment]
I've gone ahead and implemented this. Bool, Int, and Float random values are now all modules. Will be in the next release

**Comment by SouljaVR** [comment]
You are awesome @VolcanicArts Thank you! <3
